src/browser/preboot_browser.ts

```diff
@@ -22,13 +22,13 @@ import {
 } from '../preboot_interfaces';
 
 // the idea here is that preboot is a global value on the window that is used by the client
-var preboot = prebootClient();
```

Comment:
is the build system exposing this variable at all or was this unnecessary to begin with?